### PR TITLE
ENH: support multiple TR values in PARREC headers

### PR DESCRIPTION
**Pull request by grlee77**
This is a tiny modification to allow reading .PAR files with more than one repetition time listed in the header as requested in #399.

I am not entirely certain if this is a standard feature of .PAR files, or due to a custom software patch.  If it is the former, I think we should support it in nibabel.

if multiple TR values are found, a warning is issued and zooms[3] is set to the first TR listed

### CODE REVIEW

**Comment by matthew-brett** [comment]
I think we need a test - maybe an edited version of another .PAR file - if we can't get the file that caused the problem?

**Comment by grlee77** [comment]
In practice there can be more than one TR value in a pulse sequence (e.g. alternating repetition time SSFP,  MR Fingerprinting, etc)

Whether or not this is officially supported by Philips as part of the .PAR format, I don't know.
It doesn't break anything to allow multiple TR values for this field, so I am in favor of adding it.

File: nibabel/parrec.py [hunk 1]
```diff
@@ -148,7 +148,7 @@
     'Technique': ('tech',),
     'Scan resolution  (x, y)': ('scan_resolution', int, (2,)),
     'Scan mode': ('scan_mode',),
-    'Repetition time [ms]': ('repetition_time', float),
+    'Repetition time [ms]': ('repetition_time', float, None),
```
Comment (matthew-brett) [inline nibabel/parrec.py]: Would you mind adding a comment at the top of parrec above `_hdr_key_dict` noting that None means "length of vector to be determined on read"?

File: nibabel/tests/test_parrec.py [hunk 2]
```diff
@@ -777,3 +779,11 @@ def test_exts2par():
                    list(nii_img.header.extensions)):
         hdrs = exts2pars(source)
         assert_equal(len(hdrs), 2)
+
+
+def test_dualTR():
+    with open(DUAL_TR_PAR, 'rt') as fobj:
+        with suppress_warnings():
```
Comment (matthew-brett) [inline nibabel/tests/test_parrec.py]: Sorry to ask, but would you mind also checking the warning is raised?   Something like:

```
    with clear_and_catch_warnings(modules=[parrec], record=True) as wlist:
        warnings.simplefilter('always')
        dualTR_hdr = PARRECHeader.from_fileobj(fobj)
        assert_equal(len(wlist), 1)
```

**Comment by matthew-brett** [comment]
Great - thanks as ever for your patience dealing with these small things.

**Pull request merged**